README_fr.md

```diff
@@ -29,7 +39,7 @@ Suite office en ligne et collaborative, basée sur LibreOffice
 Les paramètres de Collabora sont à `https://office.domain.tld/loleaflet/dist/admin/admin.html`.
```

Comment:
The new URI for the admin is : https://office.domain.tld/browser/dist/admin/admin.html